api/controllers/game/move.js

```diff
@@ -57,7 +57,7 @@ module.exports = async function (req, res) {
     const updatedState = execute(gameState, req.body, playedBy, game.gameStates);
     const gameStateRow = await saveGamestate(updatedState);
     game.gameStates.push(gameStateRow);
-    const socketEvent = await createSocketEvent(game, updatedState);
+    const socketEvent = await createSocketEvent(game, { ...updatedState, id: gameStateRow.id });
```

Comment:
Need to get the id for the just-created gamestate from the obj saved to the db, so we spread the in-memory obj with the id from the db